static/js/user_groups.ts

```diff
@@ -195,3 +201,31 @@ export function is_user_in_group(user_group_id: number, user_id: number): boolea
     }
     return false;
 }
+
+export function get_realm_user_groups_for_dropdown_list_widget(
+    exclude_internet_group: boolean,
+    exclude_owners_group: boolean,
```

Comment:
We can probably replace these set of booleans by a list of names in future, because we may have a couple of settings where we would want to allow only moderators and above groups and options.

Reply:
Yeah, that makes sense.